plugins/linear-comparative-view/src/LinearSyntenyView/components/ImportForm.tsx

```diff
@@ -178,8 +187,8 @@ const ImportForm = observer(({ model }: { model: LinearSyntenyViewModel }) => {
 
       <Paper className={classes.formPaper}>
         <Typography style={{ textAlign: 'center' }}>
-          <b>Optional</b>: Add a .paf, .out (MashMap), .delta (Mummer), or
-          .chain file to view in the dotplot. These file types can also be
+          <b>Optional</b>: Add a .paf, .out (MashMap), .delta (Mummer), .chain,
+          or .anchors file to view in the dotplot. These file types can also be
```

Comment:
This import form should say synteny instead of dotplot, right?

Reply:
indeed, fixed! thanks